### DISCUSSION TITLE
Can't create supabase public function (non error, no loading?)

### DISCUSSION THREAD

**Discussion by jackfriks** [supabase/supabase#19804] [Questions]
I'm trying to create a new supabase function and each time i click create it just goes to a loading state and then nothing happens. I logged out, back in, and more before but nothing has seemed to change anything since yesterday. 

I get no error just a short loading state and nothing after when trying to create a function. I made the same function yesterday but deleted it to update the security from involker to definer. now i cant make any function of any type in the supabase UI.

![image](https://github.com/supabase/supabase/assets/112214002/3be83017-c48b-4206-882c-dfab1136761e)

**Comment by ssudaraka** [comment]
Hi @jackfriks, do you see any errors in your project's logs when this error happens? As a preliminary debugging step, I recommend restarting your project and see if this issue goes away.
